### ISSUE TITLE
EMSCRIPTEN version doesn't type the right keys using EN-GB keyboard

### ISSUE THREAD

**Issue by mobluse** [commanderx16/x16-emulator#189]
The EMSCRIPTEN version on https://sebastianvog.github.io/x16-emulator/x16emu.html
that is mentioned in https://github.com/commanderx16/x16-emulator/blob/master/webassembly/WebAssembly.md doesn't type the right characters using EN-GB keyboard. I press F9 until EN-GB is shown. I do the command `?CHR$($0F)`, i.e. switch to ISO-mode.

I press \` (left of 1) and ¬ (left of !, i.e. Shift+\`), but nothing is typed.
I press ' (right of ;), but it types the wrong key \` (96) instead of ' (39).
I press @ (right of :, i.e. Shift+'), but it types the wrong key ¬ instead of @.
I press # (two keys right of ;), but it types the wrong key ' instead of #.
I press ~ (two keys right of :, i.e. Shift+#), but it types the wrong key @ instead of ~.
I press \ (left of z), but it types the wrong key # instead of \.
I press | (left of Z, i.e. Shift+\), but it types the wrong key ~ instead of |.

These keys work in the native version. @Tmp2k @sebastianvog

**Comment by sebastianvog** [comment]
Sorry for late reply. I will take a look at it tonight.

**Comment by sebastianvog** [comment]
Quick update: I updated version on  https://sebastianvog.github.io/x16-emulator/x16emu.html
with the latest from master. Not sure if that made a difference.

**Comment by mobluse** [comment]
No, the problems remain. I might add that I use Chromium in Raspbian Buster Linux and keyboard layout is English (UK) and I set EN-GB using F9 in Emscripten version of x16emu. The keyboard works without problems in the native x16emu.

**Comment by Tmp2k** [comment]
There is a bug because I'm using the latest version (r33) on https://x16.tmp2k.com/ and I don't get the issue using r33 on Windows. I'll have a look into it as soon as I get chance.

**Comment by mobluse** [comment]
I've tested both mentioned sites for the Emscripten version in the most updated Chrome 77.0.3865.120 in Windows 10 and they have the same problem as the most updated Chromium in Raspbian Buster Linux. I'm typing into the blue emulator when these keys don't work at all or types wrong character. 

Another perhaps related issue: Typing into the editor on the left works, but running does not transfer all characters. If I type `` `¬'@#~\| `` in the editor only `'@#\` shows up in the editor after clicking play button. I still have EN-GB and ISO-mode `?CHR$($0F)`.
